src/de/jost_net/JVerein/gui/control/FilterControl.java

```diff
@@ -432,6 +438,49 @@ public void format(TreeItem item)
     }
   }
   
+  public TreePart getEigenschaftenAuswahlTree2(String vorbelegung,
+      boolean ohnePflicht) throws RemoteException
+  {
+    eigenschaftenAuswahlTree = new TreePart(
+        new EigenschaftenNode2(vorbelegung, ohnePflicht), null);
+    eigenschaftenAuswahlTree.addSelectionListener(
+        new EigenschaftListener2());
+    eigenschaftenAuswahlTree.setFormatter(new EigenschaftTreeFormatter2());
+    return eigenschaftenAuswahlTree;
+  }
+  
+  public static class EigenschaftTreeFormatter2 implements TreeFormatter
+  {
+
+    @Override
+    public void format(TreeItem item)
+    {
+      EigenschaftenNode2 eigenschaftitem = (EigenschaftenNode2) item.getData();
+      if (eigenschaftitem.getNodeType() == EigenschaftenNode2.ROOT
+          || eigenschaftitem
+              .getNodeType() == EigenschaftenNode2.EIGENSCHAFTGRUPPE)
+      {
+        //
+      }
```

Comment:
If-Statement hat leeren Körper. Kann refactored werden oder soll der in Naher Zukunft in einem anderen Feature dazu kommen?

Reply:
Ich habe es so übernommen, aber mit der nächsten Übergabe kommt dann was dazu. Ich werde auch hier Icons anzeigen z.B. um zu sehen ob bei Beitragsgruppen Pflicht und/oder Maxeins gesetzt ist.
Also kann man es hier so lassen.